pagopa/io-app
# feat: [IOPID-686] EmailAlreadyUsed Modal

# ⚠️ Depends on https://github.com/pagopa/io-app/pull/4946 ⚠️


## Short description
This PR adds the modal screen that informs user his email is already used. No logic, just UI.

## List of changes proposed in this pull request
- [locales/en/index.yml, locales/it/index.yml]: new labels
- ts/screens/modal/mailCheck/EmailAlreadyUsedModal.tsx: new screen

## How to test
Verify that the new screen complies with [figma](https://www.figma.com/file/oLJMcWd0SbRHDtj7WRvu7N/Univocit%C3%A0-indirizzi-email?type=design&node-id=31-12328&mode=design&t=Hm2VKPiw2EoZu827-0)
